### DISCUSSION TITLE
Separating admin and config commands?

### DISCUSSION THREAD

**Discussion by Technoguyfication** [eccentricdevotion/TARDIS#520] [Ideas]
So currently we have `/tardisadmin` (or `/tadmin` as I'll call it) to handle all administrative functions of the plugin. Having config values and admin commands under the same command can be really confusing. For example, for a new admin it may be ambiguous whether `/tadmin enter` changes an option related to TARDIS entry or enters a player's TARDIS. (Even if for us it seems obvious that it's the latter)

Would it be feasible to separate these out into two separate commands, `/tadmin` and `/tconfig`? I'd be open to creating a PR for this, but that's not a light task so I wanted to gauge how well the idea would be received first. It'd be very useful to have more admin commands, but adding these alongside the configuration commands only increases complexity. 

On the topic of more admin commands, I see this was talked about in #307, but the `tardissudo` command honestly seems like a pretty clunky way of controlling other players' TARDISes. It would probably be easier for server admins to have a collection of utilities to help players, like:

- Sending a player's TARDIS home
- Desieging a TARDIS (I know this already exists, just adding for completeness)
- Setting a TARDIS's artron level
- Configuring ARS for players
- Updating controls
- Changing chameleon presets
- Locking/unlocking doors
- etc.

If there are enough useful admin commands, then we shouldn't need tardissudo.

**Comment by eccentricdevotion** [comment]
> Locking/unlocking doors

Added to `/tardissudo` 3d6915a

**Comment by eccentricdevotion** [comment]
I'm all for this idea (in fact I started doing it tonight as I was bored) but I think there is still a need for a `/tardissudo` command as there are various admin tasks/commands that are neither player or config related.

**Comment by eccentricdevotion** [comment]
> Setting a TARDIS's artron level

`/tgive [player] artron [amount]`

**Comment by eccentricdevotion** [comment]
> Sending a player's TARDIS home

Currently possible with `/tardisremote [player] travel home`

**Reply by DalekCraft2** [reply]
You know, I believe that `/sudo` is a command in Essentials already, and it makes `/tardisremote` and `/tardissudo` unnecessary. Do you only keep those commands for servers what do not use Essentials?

**Comment by eccentricdevotion** [comment]
`/sudo` command requires the player to be online, and the `/sudo` command just forces the player to run the command.

For example `/sudo thenosefairy tardis comehere` - this won't work for an admin wanting to bring thenosefairy's TARDIS to the location that they are looking at, as the `/tardis comehere` command will use thenosefairy's target block, not the admins...

**Reply by eccentricdevotion** [reply]
Update docs - https://eccentricdevotion.github.io/TARDIS/sudo-commands.html

**Comment by Technoguyfication** [comment]
Awesome! I love the admin commands that have been added so far, I've already used quite a few of them
